tests/test_connection.py

```diff
@@ -1005,6 +1005,27 @@ def test_funding_external_wallet_corners(node_factory, bitcoind):
     # We can cancel channel after fundchannel_complete
     assert l1.rpc.fundchannel_cancel(l2.info['id'])['cancelled']
 
+    l1.rpc.connect(l2.info['id'], 'localhost', l2.port)
+    l1.rpc.fundchannel_start(l2.info['id'], amount)['funding_address']
+    assert l1.rpc.fundchannel_complete(l2.info['id'], prep['txid'], txout)['commitments_secured']
```

Comment:
The `assert` also seems a bit strange, are you trying to check that there is a `commitments_secured` element in the result? In that case an `assert 'commitments_secured' in l1.rpc.fundchannel_complete...` seems more appropriate.

Reply:
Yeah, `'commitments_secured'` is always `true`. Checking if `'commitments_secured'` field exists is more reasonable.

Reply:
but if the `commitments_secured` is ever returned as `false` then this would not have been a successful `fundchannel_complete`.

Reply:
I suppose you could instead just add a comment before this like `# A successful funding_complete will always have a commitments_secured that is true, otherwise it would have failed.`?